scheduler.rst

```diff
@@ -0,0 +1,328 @@
+Scheduler
+=========
+
+.. versionadded:: 6.3
+
+    The Scheduler component was introduced in Symfony 6.3
+
+The Symfony Scheduler is a powerful and flexible component designed to manage tasks scheduling within your PHP application.
+
+This document focuses on using the Scheduler component in the context of a full stack Symfony application.
+
+Installation
+------------
+
+In applications using :ref:`Symfony Flex <symfony-flex>`, run this command to
+install the scheduler component:
+
+.. code-block:: terminal
+
+    $ composer require symfony/scheduler
+
+Debugging the Schedule
+~~~~~~~~~~~~~~~~~~~~~~
+
+The ``debug:scheduler`` command provides a list of schedules along with their recurring messages.
+You can narrow down the list to a specific schedule.
+
+.. versionadded:: 6.4
+
+    Since version 6.4, you can even specify a date to determine the next run date using the ``--date`` option.
+    Additionally, you have the option to display terminated recurring messages using the ``--all`` option.
+
+.. code-block:: terminal
+
+    $ php bin/console debug:scheduler
+
+      Scheduler
+      =========
+
+      default
+      -------
+
+        ------------------- ------------------------- ----------------------
+        Trigger             Provider                  Next Run
+        ------------------- ------------------------- ----------------------
+        every 2 days        App\Messenger\Foo(0:17..)  Sun, 03 Dec 2023 ...
+        15 4 */3 * *        App\Messenger\Foo(0:17..)  Mon, 18 Dec 2023 ...
+       -------------------- -------------------------- ---------------------
+
+Introduction to the case
+------------------------
+
+Embarking on a task is one thing, but often, the need to repeat that task looms large.
+While one could resort to issuing commands and scheduling them with cron jobs, this approach involves external tools and additional configuration.
+
+The Scheduler component emerges as a solution, allowing you to retain control, configuration, and maintenance of task scheduling within our PHP application.
+
+At its core, the principle is straightforward: a task, considered as a message needs to be managed by a service, and this cycle must be repeated.
+Does this sound familiar? Think :doc:`Symfony Messenger docs </components/messenger>`.
+
+But while the system of Messenger proves very useful in various scenarios, there are instances where its capabilities
+fall short, particularly when dealing with repetitive tasks at regular intervals.
```

Comment:
I think the writing - while very nice - is getting long in this section. I think (but perhaps I'm wrong) that the idea of scheduling tasks is fairly easy to understand. Pointing out that this could be done by a CRON but that scheduler allows you to keep your config all in one place & is more flexible is nice. But once we have that, we can get straight to using scheduler.